js/main.js

```diff
@@ -0,0 +1,62 @@
+$(function(){
+
+  var $listEntryEl = $("[data-js='listEntry']");
+  var $todoListEl = $("[data-js='todoList']");
+  var $todoListItemEl = $("[data-js='listItem']");
+  var $selectedListItemEl = $();
+
+    var $deleteKey = false;
+    $(document).keydown(function(e){
+      if (e.keyCode === 8) {
+        $selectedListItemEl.parent().remove();
+      }
+    });
+
+$listEntryEl.on("keyup", function(e){
+     // Append a new li to the list container element
+  var $el = $(e.target);
+    if($el.val().length > 1 && e.keyCode === 13){
+       $todoListEl.prepend(`
+          <li class="todo-list__item" data-js="listItem">
+            <span class="checkmark" data-js="checkmark"> &#10003;</span>
+            <p class="todo-list__item--text" data-js="listItemText">
+          ${$el.val()}
+           </p>
+          </li>
+
+         `);
+        $el.val("");
+    var $itemInputEl = $("[data-js='itemText']");
+    var $itemCount = $($itemInputEl).length;
```

Comment:
Checking length on the wrong element here, it should testing length for a selector finding todo list items